GameFighter/Models/Person.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameFighter.Models
+{
+    public abstract class Person
+    {
+        public int Health { get; set; }
+
+        public bool IsAlive { get => Health > 0; }
```

Comment:
You can use more simplified syntax (lambda expression)

Reply:
But it looks beautiful :(